worlds/mcmillan_airfield.world

```diff
@@ -0,0 +1,49 @@
+<?xml version="1.0" ?>
+<sdf version="1.5">
+  <world name="default">
+    <scene>
+      <sky>
+        <clouds>
+          <speed>12</speed>
+        </clouds>
+      </sky>
+    </scene>
+    <!-- A global light source -->
+    <include>
+      <uri>model://sun</uri>
+    </include>
+    <include>
+      <uri>model://asphalt_plane</uri>
+    </include>
```

Comment:
Could you maybe remove the asphalt plane if it isn't necessary?

Reply:
:+1: